scripts/console.sh

```diff
@@ -11,14 +11,25 @@ if [ "$current_directory" == "scripts" ]; then
   exit 1
 fi
 
+# Change the following to the actual GPU devices you want to work on (e.g DEVICES="0,1") or to NONE if you do not plan on using any GPUs
+DEVICES="NONE"
+
+# Change the following to your caching directory if you want persistent caching (e.g. CACHE_DIR="/scratch1/username/.cache"), else set it to NONE
+# if you plan on mounting a cache-folder you will have to create one, before you can run this script
+CACHE_DIR="NONE"
+
+# Change the following image-tag to the name of your own image, if you do not want to use the default one
+IMAGE_TAG="konstantinjdobler/nlp-research-template"
+
 docker run -it \
-    --gpus='device=CHANGE_ME' \
+    --user $(id -u):$(id -g) \
+    $([[ "$DEVICES" != "NONE" ]] && echo "--gpus=\"device=$DEVICES\"") \
```

Comment:
```suggestion
    $([[ "$DEVICES" != "NONE" ]] && echo "--gpus device=$DEVICES") \
```

Did this work before like it is in the PR currently/ According to the docs, you have to use `--gpus='"device=$DEVICES"'` or `--gpus device=$DEVICES`

Reply:
It is working this way. I tested selecting only one, multiple or no GPUs.